src/forklift/cli.py

```diff
@@ -379,3 +387,29 @@ def speedtest(pallet_location):
     print('{1}Dry Run Output{0}{2}{3}'.format(Fore.RESET, Fore.CYAN, linesep, dry_report))
     print('{1}Repeat Run Output{0}{2}{3}'.format(Fore.RESET, Fore.CYAN, linesep, repeat_report))
     print('{3}{0}{1}Speed Test Results{3}{0}{2}Dry Run:{0} {4}{3}{2}Repeat:{0} {5}'.format(Fore.RESET, Fore.GREEN, Fore.CYAN, linesep, dry_run, repeat))
+
+
+def update_static(file_path):
+    log.info('updating/creating static data for pallet(s) in %s', file_path)
+
+    start_seconds = clock()
+
+    git_errors = git_update()
```

Comment:
this looks like it should belong earlier in the process. Is it going to happen twice on a normal lift run now? Or is this only run on when invoked specifically from the cli?